openedx/core/djangoapps/discussions/config/waffle.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module contains various configuration settings via
+waffle switches for the discussions app.
+"""
+
+from edx_toggles.toggles import LegacyWaffleFlagNamespace
+
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag
+
+
+WAFFLE_DISCUSSION_LEGACY_SETTINGS_NAMESPACE = LegacyWaffleFlagNamespace(name='discussions')
+
+# .. toggle_name: discussions.override_discussion_legacy_settings
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to override visibility settings for discussion settings in course advance settings.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-06-15
+# .. toggle_target_removal_date: 2021-12-31
```

Comment:
This is random?

Reply:
sort of, estimated time till we remove the flag!